pipeline/metadata/5.1.yaml

```diff
@@ -0,0 +1,489 @@
+# =====================================================================================
+# Metadata file for 5.1 RHCS release version execution for pipeline V3.
+# Single file to execute the all tier-1, tier-2, tier-3 and tier-4 suites in parallel
+# as per defined in each stages.
+# Each stage will execute in sequential pattern.
+# Contains all Default parameter used for Execution when no override is specified.
+# parameter requied to create a similar custom file is suite name, suite yaml file, global configuration file,
+# platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
+# =====================================================================================
+- name: "test-bootstrap-custom-ssl-dashboard-port-and-apply-spec"
+  suite: "suites/pacific/cephadm/tier-1_ssl_dashboard_port.yaml"
+  global-conf: "conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.1"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - cephadm
+    - stage-1
+- name: "test-bootstrap-skip-dashboard-and-custom-ceph-directory-organisation"
+  suite: "suites/pacific/cephadm/tier-1_skip_dashboard.yaml"
+  global-conf: "conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.1"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - cephadm
+    - stage-1
+- name: "test-cephadm-apply-service-spec"
+  suite: "suites/pacific/cephadm/tier-1_5-1_service-apply-spec.yaml"
+  global-conf: "conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.1"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - cephadm
+    - stage-1
+- name: "test-cephadm-upgrade-build-to-build-5x"
+  suite: "suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml"
+  global-conf: "conf/pacific/upgrades/tier-1_upgrade_cephadm.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.1"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - upgrades
```

Comment:
Upgrades is not an FG in itself. So this test suite can contain dmfg also as another tag along with upgrades to all upgrade test suites.

Reply:
added both FG as
-  upgrades
-  dmfg